server/aws-lsp-codewhisperer/src/language-server/codeWhispererServer.ts

```diff
@@ -259,18 +259,6 @@ const mergeSuggestionsWithRightContext = (
     })
 }
 
-// Checks if any suggestion in list of suggestions matches with left context of the file
-const hasLeftContextMatch = (suggestions: Suggestion[], leftFileContent: string): boolean => {
```

Comment:
This change is not related to the chat server but noticed this unused function and removed it